README.md

```diff
@@ -1,2 +1,41 @@
-# methodology
-Measurement methodology for advertising emissions
+# An open framework for measurement advertising emissions
+Our goal with this project is to build a framework where the media and advertising industry can collaborate on best practices for measuring emissions from the advertising value chain. This project was originally developed by Scope3, and is the foundation of the v2 Scope3 methodology.
+
+Measuring emissions is extremely complicated in general. In the words of one industry leader, "it took us 100 years to figure out how to do financial accounting... and now we're trying to figure out carbon accounting in 2 or 3." As such, we feel like it's critical to learn in public and to be honest about what we know and what we don't know. Assuming the carbon accounting will require the same auditing and assurance process as the financial accounting world, we hope that this project will enable every step of the process to be traced and validated.
+
+Our measurement process, at a high level, works as follows:
+1. Gather public sustainability materials from industry participants
```

Comment:
Why materials not documents or reports?

I’m also wondering if we should expand this, because we may not be just getting sustainability reports but potentially also annual reports/DEI reports which will give us other information points like employee counts and those are not necessarily sustainability focused at all. which may not be purely sustainability focused.

Reply:
great point - will update next doc rev